contracts/BaseStrategy.sol

```diff
@@ -467,11 +470,6 @@ abstract contract BaseStrategy {
      *   See `vault.report()` for further details.
      */
     function distributeRewards() internal virtual {
-        // Transfer 100% of newly-minted shares awarded to this contract to the rewards address.
-        uint256 balance = vault.balanceOf(address(this));
-        if (balance > 0) {
-            vault.transfer(rewards, balance);
-        }
     }
```

Comment:
shoudl we remove this function entirely then?

Reply:
yes i think so

Reply:
let's do it